Example/RxRealm/ViewController.swift

```diff
@@ -32,23 +35,28 @@ class ViewController: UIViewController {
         //Observable<Array<Lap>>
         let laps = realm.objects(Lap).sorted("time", ascending: false).asObservableArray()
         
-        laps.map {array in
-            return array.prefix(3) //get array slice of the 3 most recent items
-        }
+        laps
         .bindTo(tableView.rx_itemsWithCellIdentifier("Cell", cellType: UITableViewCell.self)) {row, element, cell in
             cell.textLabel!.text = formatter.stringFromDate(NSDate(timeIntervalSinceReferenceDate: element.time))
         }.addDisposableTo(bag)
 
-        //start adding laps
-        addLap()
-    }
-    
-    func addLap() {
-        let realm = try! Realm()
-        try! realm.write {
-            realm.add(Lap())
-        }
+        
+        addOneItemButton.rx_tap
+            .map {
+                return Lap()
+            }
+            .bindTo(realm.rx_addObject())
```

Comment:
Yep, I totally like `var` better. And I've implemented it like this at first, and demo App compiles Ok, 
But when I wanted to implement tests and Swift compiler said: " NO, NO, NO" 😔

<img width="1125" alt="screen shot 2016-05-29 at 3 40 47 pm" src="https://cloud.githubusercontent.com/assets/4622322/15633372/21b7bf96-25b4-11e6-8f40-ac3c21d05829.png">

There problem also, looks like in Swift there is no way to make generic `var`
So I can't not do like this  😔 :

```Swift
    public var rx_addObject: AnyObserver<O:Object> {
        return AnyObserver { [weak self] event in
            switch event {
            case .Next(let object):
                try! self?.write {
                    self?.add(object)
                }
            default:
                break
            }
        }
    }
```